test/swivel/Interfaces.sol

```diff
@@ -60,6 +60,12 @@ interface IEuler {
   function withdraw(uint256, uint256) external; // void
 }
 
+interface ILido {
+  function wrap(uint256) external returns (uint256);
+  function unwrap(uint256) external returns (uint256);
+  function getStETHByWstETH(uint256) external returns (uint256);
+}
+
```

Comment:
Re: getting stETH address, it can be retrieved using `stETH`.

See the method here:
https://etherscan.io/token/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0#readContract#F9

Reply:
yeah, as long as that is the equivalent of `underlying` then 100% going with this

Reply:
Yea agree. I missed this on my look through and its way more consistent w/ our pattern

Reply:
ooohh. it's a public variable. that's why we didn't see it in the method list